api/src/test/java/org/pmiops/workbench/elasticsearch/ElasticFiltersTest.java

```diff
@@ -800,13 +828,15 @@ public void testAgeAndVisitQuery() {
     SearchParameter ethParam =
         new SearchParameter()
             .conceptId(Long.parseLong(conceptId))
-            .type(TreeType.DEMO.toString())
-            .subtype(TreeSubType.ETH.toString())
```

Comment:
Is subtype deprecated with universal search? Are we tracking removal somewhere?

Reply:
subtype is still relevant for some trees(Measurement, Physical Measurement and Surveys(PPI)). Referencing it here: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/util/CriteriaLookupUtil.java#L226
TreeType and TreeSubType are being replace with CriteriaType and CriteriaSubType in the new CB search. Tracking removal here: https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/cb_search_api.yaml#L780